work/worker.go

```diff
@@ -485,24 +485,22 @@ func (self *worker) commitNewWork() {
 	self.currentMu.Lock()
 	defer self.currentMu.Unlock()
 
-	tstart := time.Now()
 	parent := self.chain.CurrentBlock()
 
 	// TODO-Klaytn drop or missing tx
+	tstart := time.Now()
```

Comment:
Can I ask why you move `tstart := time.Now()`?

Reply:
@yoomee1313 I just want to assign and re-assign `tstart` and `tstamp` in sequencely. It's not a big deal.

Reply:
I understood.. :)